UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/AzureBlobStorageService.cs

```diff
@@ -100,6 +114,39 @@ public async Task<SalesCatalogueProductResponse> DownloadSalesCatalogueResponse(
             logger.LogInformation(EventIds.DownloadSalesCatalogueResponsDataCompleted.ToEventId(), "Sales catalogue response download completed from blob for the scsResponseUri:{scsResponseUri} and _X-Correlation-ID:{correlationId}", scsResponseUri, correlationId);
             return salesCatalogueProductResponse;
         }
-       
+
+
+        private (int, string) GetInstanceCountBasedOnFileSize(double fileSizeInMB)
+        {
+            if (fileSizeInMB > storageConfig.Value.SmallExchangeSetSizeInMB &&
+                fileSizeInMB <= storageConfig.Value.LargeExchangeSetSizeInMB)
+            {
+                return (mediumExchangeSetInstance.GetInstanceCount(storageConfig.Value.MediumExchangeSetInstance), ExchangeSetType.MediumExchangeSet.ToString());
+            }
+            else if (fileSizeInMB > storageConfig.Value.LargeExchangeSetSizeInMB)
+            {
+                return (largeExchangeSetInstance.GetInstanceCount(storageConfig.Value.LargeExchangeSetInstance), ExchangeSetType.LargeExchangeSetInstance.ToString());
+            }
+            else
+            {
+                return (smallExchangeSetInstance.GetInstanceCount(storageConfig.Value.MediumExchangeSetInstance), ExchangeSetType.SmallExchangeSet.ToString());
```

Comment:
Kindly change (storageConfig.Value.MediumExchangeSetInstance) to (storageConfig.Value.SmallExchangeSetInstance)

Reply:
Yes updated accordingly.